app/src/main/java/com/github/muellerma/tabletoptools/ui/fragments/CountDownFragment.kt

```diff
@@ -12,22 +13,59 @@ import android.view.View
 import android.view.ViewGroup
 import android.widget.Button
 import android.widget.TextView
+import androidx.core.content.edit
 import androidx.core.view.isInvisible
 import androidx.core.view.isVisible
-import androidx.fragment.app.Fragment
 import com.github.muellerma.tabletoptools.R
+import com.github.muellerma.tabletoptools.ui.dialog.CountDownPickerDialog
+import java.util.concurrent.TimeUnit
 
 
-class FMTFragment : AbstractBaseFragment() {
+class CountDownFragment : AbstractBaseFragment() {
     private lateinit var timerView1: TextView
     private lateinit var timerView2: TextView
     private lateinit var startButton: Button
+    private lateinit var editButton: Button
     private lateinit var resetButton: Button
     private var timer: CountDownTimer? = null
-    private var remainingTime: Long = DEFAULT_TIMER
+    var originalTime: Long = DEFAULT_TIMER
+    private var remainingTime: Long = originalTime
     private var timerRunning = false
     private var player: MediaPlayer? = null
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val sharedPref = activity?.getPreferences(Context.MODE_PRIVATE) ?: return
```

Comment:
Can you create an extension function in a new file in the utils package: `fun Context.preferences()` that returns a shared preferences instance? It can be used across all fragments to ensure they all use the same preferences file.
Also: I would default to 5 minutes if `sharedPref` is null.